lib/lunchbot/lunchbot_data.ex

```diff
@@ -586,46 +586,43 @@ defmodule Lunchbot.LunchbotData do
     end
   end
 
-  def format_orders(order_struct) do
-    if !is_nil(order_struct) do
-      orders =
-        for order <- order_struct do
-          timestamp = order.inserted_at
-          user = order.user
-
-          order_items =
-            for order_item <- order.order_items do
-              item_with_options =
-                for order_item_option <- order_item.options do
-                  order_item_option
-                end
-                |> Enum.join(", ")
-
-              if String.length(item_with_options) > 0 do
-                order_item.item <>
-                  " w/ options: " <>
-                  item_with_options
-              else
-                order_item.item
-              end
-            end
+  def format_orders(order_struct = [%Order{} | _]) do
```

Comment:
Would it be feasible to use a default param here rather than pattern matching?  They do slightly different things, so depending on how this function is called one might be a better option than the other.  Something like:

`def format_orders(order_struct \\ [%Order{}]) do`